### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by 1a**
- bootstrapped the app
- implemented the basic header and footer

### CODE REVIEW

File: src/index.ejs [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+  <script src="https://cdn.jsdelivr.net/npm/systemjs@6.8.3/dist/system.js"></script>
+  <script src="https://cdn.jsdelivr.net/npm/systemjs@6.8.3/dist/extras/amd.js"></script>
+  <% } else { %>
+  <script src="https://cdn.jsdelivr.net/npm/systemjs@6.8.3/dist/system.min.js"></script>
+  <script src="https://cdn.jsdelivr.net/npm/systemjs@6.8.3/dist/extras/amd.min.js"></script>
+  <% } %>
+
+  <template id="single-spa-layout">
+    <single-spa-router containerEl=".mtfh-container">
+      <!-- <nav>
```
Comment (rocco-scaramuzzi) [inline src/index.ejs:75]: let's remove those comments if we don't need them

Comment (1a) [reply src/index.ejs:75]: good spot. wilco

Comment (1a) [reply src/index.ejs]: done

File: src/index.ejs [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+<body>
+  <div class="mtfh">
+    <div class="mtfh-header">
+      <header class="lbh-header  ">
+        <div class="lbh-header__main">
+          <div class="lbh-container lbh-header__wrapper ">
+            <div class="mtfh-header__section">
+              <h1 class="lbh-header__title">
+                <a href="/" class="lbh-header__title-link">
+                  <svg
```
[the remark this answers is not in the document]
Comment (1a) [reply src/index.ejs:95]: good idea 👍, I will create this as a separate task

**Review by zivc — APPROVED** [review]

File: src/index.ejs (hunk 2 above)
[the remark this answers is not in the document]
Comment (1a) [reply src/index.ejs:95]: see MTTL-461

**Pull request merged**